Applications/AlgorithmPlaza.qmd

```diff
@@ -5,108 +5,112 @@ aliases:
+Each registered user acts as an individual organisation; profile details can be modified, as discussed earlier in Step 1. However, step 2 could be helpful when working with multiple individuals or being part of multiple organisations. Organisations allow collaboration with other users, facilitating the sharing of billing and service accounts. For more information on managing the organisation, please refer to the [organisation's documentation](PlazaDetails/ManageOrg.qmd) of the openEO Algorithm Plaza.
 
 ###### Step 3: Check your Credits
 
-Using any openEO processes, including those offered as services in this marketplace, consumes a certain amount of credits. Credits are crucial in executing openEO processes, serving as the main currency for accessing services and processing resources. Notably, these credits are shared among organisations. Whenever a service or a supported processing platform is executed, credits from the shared pool cover the resource consumption.
+Using any openEO processes, including those offered as services in this marketplace, consumes a certain amount of credits. Notably, these credits are shared among the organisation's members. Credits are consumed whenever a service or a supported openEO process is used.
 
-This marketplace simplifies credit management, allowing users to monitor their account's credits easily. You can check your openEO credit under the [`Billing`](https://marketplace-portal.dataspace.copernicus.eu/billing){target="_blank"} section. Moreover, every user is provided with 4000 credits each month, with which they can execute multiple services.
+This marketplace simplifies credit management, allowing users to monitor their accounts. The credit can be monitored under the [`Billing`](https://marketplace-portal.dataspace.copernicus.eu/billing){target="_blank"} section. Moreover, every user is provided with 4000 free openEO credits each month to execute multiple services.
 
 
 ![](_images/billing2.png)
 
-Credits are deducted based on the chosen services and spatial extent.The amount will vary depending on the processing complexity and time required for each type of service. Detailed examples of some well-known services and how they fit into the 4000 free credits can further be found [here](PlazaDetails/Strength.qmd).
+Credits are deducted based on the chosen services. The amount will vary depending on the processing complexity and CPU time required for each type of service. Detailed examples of the credit usage of well-known services can be found [here](PlazaDetails/Strength.qmd).
```

Comment:
It is not necessarily only cpu time, even when the instance is doing nothing you pay. Maybe good to verify with Jeroen D